apps/islamic-website/app/robots.ts

```diff
@@ -0,0 +1,18 @@
+import { MetadataRoute } from 'next';
+import { SITE_URL } from '../constants';
+import { SUPPORTED_LOCALES } from '../constants';
+
+export default function robots(): MetadataRoute.Robots {
+  const sitemapUrls = SUPPORTED_LOCALES.map((locale) => {
+    return new URL(`/${locale}/sitemap.xml`, SITE_URL).toString();
+  });
+
+  return {
+    rules: {
+      userAgent: '*',
+      allow: '/',
+    },
+    sitemap: sitemapUrls,
```

Comment:
here should be url to single sitemap, cuz you move it to `app` folder

Reply:
Done